go-controller/pkg/cni/helper_linux.go

```diff
@@ -248,6 +248,9 @@ func (pr *PodRequest) ConfigureInterface(namespace string, podName string, ifInf
 	// but only the latest one should have the iface-id set.
 	uuids, _ := ovsFind("Interface", "_uuid", "external-ids:iface-id="+ifaceID)
 	for _, uuid := range uuids {
+		if len(uuid) == 0 || uuid == `""` {
+			continue
```

Comment:
This shouldn't be necessary; the function should not be returning bad data...

Reply:
@danwinship Agree